app/design/frontend/base/default/template/boltpay/catalog/product/configure_checkout.phtml

```diff
@@ -17,94 +17,57 @@
 
 /* @var $this Bolt_Boltpay_Block_Catalog_Product_Boltpay */
 ?>
-<?php if ($this->isEnabledProductPageCheckout() && $this->isSupportedProductType()): ?>
-<script type="text/javascript">
-var productPageCheckoutSelector = '<?php $this->escapeHtml($this->getProductPageCheckoutSelector()) ?>';
-var order_completed = false;
-var do_checks = 1;
-<?php $productTierPrice = $this->getProductTierPrice(); ?>
```

Comment:
It looks like you're removing the logic for adding multiple quantities and tier pricing and I don't see where this is being added back. Can you confirm it works in those cases and creates a quick video showing that it works?

Reply:
@johnmccombs1 http://ayemelyanov.brightgrove.kharkov.ua:81/accessories/bags-luggage/classic-hardshell-suitcase-484.html

Reply:
@johnmccombs1 i used exisiting logic from Magento: VarienForm and spConfig

Reply:
http://ayemelyanov.brightgrove.kharkov.ua:81/accessories/bags-luggage/classic-hardshell-suitcase-484.html
Checked on this product with @YevhenShBolt - works fine

Reply:
@oldDriver I've just found a bug in the simple product page here http://ayemelyanov.brightgrove.kharkov.ua:81/accessories/bags-luggage/isla-crossbody-handbag.html

If you add '2' as the quantity and then click on the Buy Now button, it only shows '1' as the quantity on the Bolt modal. I believe this breaks the tier price as well.

Reply:
@johnmccombs1 For existing functionality qty = 1 was hardcoded.
But i could add functionality on change quantity for simple products.
Does it make sense?

Reply:
@oldDriver The existing functionality already worked like that.  See a live demo page here:  https://www.headlightsdepot.com/perde-chrome-projector-headlight-set-for-2006-2010-volkswagen-beetle-with-factory-equipped-halogen-headlamps-sku-vw00502-hlap.html

Reply:
@johnmccombs1 please check http://ayemelyanov.brightgrove.kharkov.ua:81/accessories/eyewear/jackie-o-round-sunglasses.html (i added tier prices for testing)

Reply:
@johnmccombs1 fixed this, please take a look
http://ayemelyanov.brightgrove.kharkov.ua:81/accessories/eyewear/jackie-o-round-sunglasses.html

Reply:
@oldDriver Current steps:
1. Load the page 
2. The Bolt checkout button is rendered
3. Click on the Bolt button
4. Enter the shipping address

New steps:
1. Load the page
2. A default Buy Now button (not Bolt checkout button) is rendered
3. Click on the default Buy Now button
4. An Ajax request is sent to the server to get a token for the order
5. The Bolt checkout button is rendered based on the token
6. Enter the shipping address


=> There is an issue with the new steps as it creates extra steps which don't give the customer a good user experience for the Bolt Buy Now button. The Bolt Buy now button should always be ready on the PDP as it's documented in the doc here https://screencast.com/t/Gr2FueC0TziX